### PR TITLE
Pipeliner CRM

### PR DESCRIPTION
**Pull request by vaclavcmar**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by vaclavcmar** [comment]
<!--
/* Font Definitions */
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h3
	{mso-style-priority:9;
	mso-style-link:"Heading 3 Char";
	mso-margin-top-alt:auto;
	margin-right:0in;
	mso-margin-bottom-alt:auto;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
code
	{mso-style-priority:99;
	font-family:"Courier New";}
pre
	{mso-style-priority:99;
	mso-style-link:"HTML Preformatted Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Courier New";}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-priority:9;
	mso-style-link:"Heading 3";
	font-family:"Calibri",sans-serif;
	font-weight:bold;}
span.HTMLPreformattedChar
	{mso-style-name:"HTML Preformatted Char";
	mso-style-priority:99;
	mso-style-link:"HTML Preformatted";
	font-family:"Courier New";}
.MsoChpDefault
	{mso-style-type:export-only;
	font-family:"Calibri",sans-serif;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
/* List Definitions */
@list l0
	{mso-list-id:503862897;
	mso-list-template-ids:-1;}
@list l1
	{mso-list-id:623120277;
	mso-list-template-ids:-1;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l1:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l1:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level5
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level8
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l1:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l2
	{mso-list-id:1014108580;
	mso-list-template-ids:-1;}
@list l2:level1
	{mso-level-start-at:6;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3
	{mso-list-id:2015644794;
	mso-list-template-ids:-1;}
@list l3:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l3:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l3:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level5
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level8
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l3:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4
	{mso-list-id:2133555822;
	mso-list-template-ids:-1;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l4:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l4:level3
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level4
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level5
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level6
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level7
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level8
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
@list l4:level9
	{mso-level-number-format:bullet;
	mso-level-text:\F0A7;
	mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Wingdings;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
***@***.*** agree company="Pipelinersales, Inc." Vaclav Cmar From: microsoft-github-policy-service[bot]Sent: Thursday, July 13, 2023 5:58 PMTo: microsoft/PowerPlatformConnectorsCc: vaclavcmar; MentionSubject: Re: [microsoft/PowerPlatformConnectors] Pipeliner CRM (PR ***@***.*** please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following ***@***.*** agree [company="{your company}"]Options:(default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my ***@***.*** agree(when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my ***@***.*** agree company="Microsoft"Contributor License Agreement Contribution License AgreementThis Contribution License Agreement (“Agreement”) is agreed to by the party signing below (“You”),and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Yourcontributions to Microsoft open source projects. This Agreement is effective as of the latest signaturedate below.Definitions.“Code” means the computer software code, whether in human-readable or machine-executable form,that is delivered by You to Microsoft under this Agreement.“Project” means any of the projects owned or managed by Microsoft and offered under a licenseapproved by the Open Source Initiative (www.opensource.org).“Submit” is the act of uploading, submitting, transmitting, or distributing code or other content to anyProject, including but not limited to communication on electronic mailing lists, source code controlsystems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose ofdiscussing and improving that Project, but excluding communication that is conspicuously marked orotherwise designated in writing by You as “Not a Submission.”“Submission” means the Code and any other copyrightable material Submitted by You, including anyassociated comments and documentation.Your Submission. You must agree to the terms of this Agreement before making a Submission to anyProject. This Agreement covers any and all Submissions that You, now or in the future (except asdescribed in Section 4 below), Submit to any Project.Originality of Work. You represent that each of Your Submissions is entirely Your original work.Should You wish to Submit materials that are not Your original work, You may Submit them separatelyto the Project if You (a) retain all copyright and license information that was in the materials as Youreceived them, (b) in the description accompanying Your Submission, include the phrase “Submissioncontaining materials of a third party:” followed by the names of the third party and any licenses or otherrestrictions of which You are aware, and (c) follow any other instructions in the Project’s writtenguidelines concerning Submissions.Your Employer. References to “employer” in this Agreement include Your employer or anyone elsefor whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If YourSubmission is made in the course of Your work for an employer or Your employer has intellectualproperty rights in Your Submission by contract or applicable law, You must secure permission from Youremployer to make the Submission before signing this Agreement. In that case, the term “You” in thisAgreement will refer to You and the employer collectively. If You change employers in the future anddesire to Submit additional Submissions for the new employer, then You agree to sign a new Agreementand secure permission from the new employer before Submitting those Submissions.Licenses.Copyright License. You grant Microsoft, and those who receive the Submission directly orindirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in theSubmission to reproduce, prepare derivative works of, publicly display, publicly perform, and distributethe Submission and such derivative works, and to sublicense any or all of the foregoing rights to thirdparties.Patent License. You grant Microsoft, and those who receive the Submission directly orindirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license underYour patent claims that are necessarily infringed by the Submission or the combination of theSubmission with the Project to which it was Submitted to make, have made, use, offer to sell, sell andimport or otherwise dispose of the Submission alone or with the Project.Other Rights Reserved. Each party reserves all rights not expressly granted in this Agreement.No additional licenses or rights whatsoever (including, without limitation, any implied licenses) aregranted by implication, exhaustion, estoppel or otherwise.Representations and Warranties. You represent that You are legally entitled to grant the abovelicenses. You represent that each of Your Submissions is entirely Your original work (except as You mayhave disclosed under Section 3). You represent that You have secured permission from Your employer tomake the Submission in cases where Your Submission is made in the course of Your work for Youremployer or Your employer has intellectual property rights in Your Submission by contract or applicablelaw. If You are signing this Agreement on behalf of Your employer, You represent and warrant that Youhave the necessary authority to bind the listed employer to the obligations contained in this Agreement.You are not expected to provide support for Your Submission, unless You choose to do so. UNLESSREQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIESEXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT ISPROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OFNONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.Notice to Microsoft. You agree to notify Microsoft in writing of any facts or circumstances of whichYou later become aware that would make Your representations in this Agreement inaccurate in anyrespect.Information about Submissions. You agree that contributions to Projects and information aboutcontributions may be maintained indefinitely and disclosed publicly, including Your name and otherinformation that You submit with Your Submission.Governing Law/Jurisdiction. This Agreement is governed by the laws of the State of Washington, andthe parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent toexclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive alldefenses of lack of personal jurisdiction and forum non-conveniens.Entire Agreement/Assignment. This Agreement is the entire agreement between the parties, andsupersedes any and all prior agreements, understandings or communications, written or oral, betweenthe parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by vaclavcmar** [comment]
@microsoft-github-policy-service agree company="Pipelinersales, Inc."

Vaclav Cmar

**Comment by vmanoharas** [comment]
Hello @vaclavcmar, 

There was validation issue, currently we have resolved them.  Please resubmit/commit your PR.

File: certified-connectors/Pipeliner CRM/Readme.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Pipeliner CRM
+Pipeliner is a next generation sales CRM software designed to empower sales teams and grow profitable customer relationships. The conector provides an API to work with Pipeliner records.
+
```
Comment (vmanoharas) [inline certified-connectors/Pipeliner CRM/Readme.md:3]: Hello @vaclavcmar 

Please refer this readme.md file template link and update your readme.md file duly. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by vmanoharas — COMMENTED** [review]
Please resolve review comments.

**Review by vmanoharas — COMMENTED** [review]
Hello @vaclavcmar, 

I see that you updated readme.md file. Thanks!

**Review by vmanoharas — APPROVED** [review]
Hello @vaclavcmar

I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**